test/test_link.rb

```diff
@@ -40,6 +40,27 @@ module SDF
                 assert Eigen::Quaternion.Identity.approx?(i.pose.rotation)
                 assert Eigen::MatrixX.from_a([1,0,0, 0,1,0, 0,0,1], 3, 3, false), i.inertia
             end
+
+            it "returns a true kinematic link" do
```

Comment:
The best for a particular method is to add a describe block for the method itself.

Moreover, try to get better descriptions in the 'it' string

```
describe "#kinematic?" do
   it "returns false if the link is explicitely set as not kinematic" do
   end
   it "returns true if the link is explicitly set as kinematic" do
   end
   it "returns false if the link does not have an explicit kinematic element" do
   end
end
```